test/strategies/Strategy.test.ts

```diff
@@ -150,7 +162,6 @@ export function testStrategy(
 
     testStrategyAccessControl()
     testStrategyDeposit()
-    testStrategyERC165()
```

Comment:
This test is a pain in the neck, it adds little value and requires frequent mainenance. Let's re-enable it after the interface becomes more stable.